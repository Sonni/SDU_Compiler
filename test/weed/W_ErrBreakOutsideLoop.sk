func f(): int
    if true then
        break;

    return 42;
end f

write f();