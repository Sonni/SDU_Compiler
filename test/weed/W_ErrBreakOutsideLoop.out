Weeder should output an error
